docs/manual/java/guide/services/ServiceErrorHandling.md

```diff
@@ -9,6 +9,9 @@ There are a number of principles behind the design of Lagom's built in error han
 * If possible, Lagom will try to reconstruct errors on the client side when thrown on the service side.  So, if the server side throws an exception saying it couldn't serialize something, the client code should receive that same exception.
 * If possible, exceptions should be mapped to idiomatic protocol response codes, such as HTTP 4xx and 5xx status codes and WebSocket error close codes.
 
+
+If you are using Lagom to consume a service (either implemented in Lagom or a third-party stack) the client Lagom provides will map 4xx and 5xx to exceptions in all cases. That has an impact on the [[Circuit Breakers|ServiceClients#Circuit-Breakers]] the client is using to connect to that endpoint. By default Lagom Circuit Breakers will account any exception as an error but that behavior is [[configurable|ServiceClients#Circuit-Breaker-Configuration]]. So 4xx and 5xx will be mapped to exceptions but you can whitelist what exceptions should not trip the circuit breaker.
```

Comment:
"will map 4xx and 5xx HTTP codes"? 
Also suggest the following simplification of the wording:
If you are using Lagom to consume a Lagom or third-party service, the client Lagom provides maps 4xx and 5xx to exceptions in all cases. The default behavior of [[Circuit Breakers|ServiceClients#Circuit-Breakers]] is to interpret exceptions as errors.   You can  [[whitelist|ServiceClients#Circuit-Breaker-Configuration]] specific 4xx and 5xx exceptions, if you do not want them to trip the circuit breaker.

Reply:
Thanks @rstento but I think this suggestion may add confusion. Users that are still unfamiliar with the LagomClient-vs-CircuitBreaker relation may not understand the sentence as you suggested. In my wording, I stated 4xx/5xx are mapped to exceptions by the client, then exceptions will impact the circuit breakers the client uses, finally I briefly explain what the impact is and link to the docs.

Well, that was my intent, if that was not clear I need to improve it. :-)